scenes/test_object.txt

```diff
@@ -3,15 +3,28 @@
 *temp test_count 0
 *temp tests_passed 0
 
+*comment test data (see startup.txt)
+*comment dragon_field_1 "name"
+*comment dragon_field_2 "hit_points"
+*comment dragon_field_max 2
+*comment dragon_name ""
+*comment dragon_hit_points ""
+*comment dragon_1_name "Calvin"
+*comment dragon_1_hit_points 100
+*comment dragon_2_name "Hobbes"
+*comment dragon_2_hit_points 150
+
+
 *gosub test_start "get unmodified data"
-*gosub_scene cslib_object get "testobj" 1
-*gosub test_assert (testobj_testfield1 = testobj_1_testfield1) "=> Expected - ${testobj_1_testfield1}[n/]=> Received - ${testobj_testfield1}[n/]"
+*gosub_scene cslib_object get "dragon" 1
+*gosub test_assert (dragon_name = dragon_1_name) "=> Expected - ${dragon_1_name}[n/]=> Received - ${dragon_name}[n/]"
 *gosub test_finish
 
 *gosub test_start "set (modify) and get (modified) data"
-*gosub_scene cslib_object set "testobj" 2 "testfield1" "Locke"
-*gosub_scene cslib_object get "testobj" 2
-*gosub test_assert ((testobj_2_testfield1 = "Locke") and (testobj_testfield1 = testobj_2_testfield1)) "=> Expected - Locke[n/]=> Received - ${testobj_testfield1}[n/]"
+*gosub_scene cslib_object set "dragon" 2 "name" "Locke"
```

Comment:
2 of 2 last requests: assert that name isn't _already_ "Locke" before setting it.

Reply:
Doh! Good point. I'll try and wrap this up later today (Sydney time)